lib/std/c/builtins.zig

```diff
@@ -6,70 +6,70 @@
 
 const std = @import("std");
 
-pub inline fn __builtin_bswap16(val: u16) callconv(.C) u16 { return @byteSwap(u16, val); }
-pub inline fn __builtin_bswap32(val: u32) callconv(.C) u32 { return @byteSwap(u32, val); }
-pub inline fn __builtin_bswap64(val: u64) callconv(.C) u64 { return @byteSwap(u64, val); }
+pub fn __builtin_bswap16(val: u16) callconv(.C) u16 { return @byteSwap(u16, val); }
```

Comment:
Shouldn't these be changed to `callconv(.Inline)`?

Reply:
This highlights a problem I have with this change, having inline be a calling convention forces all inline functions to have zig's standard calling convention.

If you specifically need a particular calling convention as in this case it makes in impossible to mark the function inline.

Reply:
> Shouldn't these be changed to callconv(.Inline)?

I thought these were exported as ABI, but these functions seem to be actually used by Zig code translated from C, so yes.

> This highlights a problem I have with this change, having inline be a calling convention forces all inline functions to have zig's standard calling convention.

In Zig, `inline` doesn't mean the same thing as in C, it means always inline, with a compile error if it's not possible.
I don't think it makes sense to have both a calling convention and an inline modifier, since the calling convention will never be used in the final program.

Reply:
On second thought your right, ignore me.